pyccel/codegen/printing/cwrappercode.py

```diff
@@ -1,5 +1,7 @@
 # coding: utf-8
 # pylint: disable=R0201
+# pylint: disable=wildcard-import
```

Comment:
We would like to avoid disabling errors in the `pyccel` folder. I know that `from pyccel.errors.messages import *` is used in several files, but the long term aim is to remove this syntax not to hide errors if this syntax is used for other imports.

If you want to disable a pytest error for 1 line only rather than the whole file you can add a comment to the end of that line

In this case though I think that there are few enough messages used that we can stop using this syntax. The only object that need to be imported from errors.messages is:
- PYCCEL_RESTRICTION_TODO

Reply:
done.